spec/requests/admins_controller.rb

```diff
@@ -0,0 +1,199 @@
+# frozen_string_literal: true
```

Comment:
Refactor this file try with https://devhints.io/factory_bot

Reply:
Add _spec to file name, rspec can't scan this file and run test.
<img width="744" alt="Screen Shot 2023-05-04 at 11 07 01" src="https://user-images.githubusercontent.com/115602568/236110594-08824baa-e126-42c7-9c12-c9ecc2c8a575.png">